### PR TITLE
[API] Console Application for Calculating Travel Time

### PR DESCRIPTION
**Pull request by RoqayaAlmamari**

### CODE REVIEW

File: src/main/java/org/example/GoogleDistanceMatrixAPI.java [hunk 1]
```diff
@@ -1,54 +1,58 @@
...
 import java.util.Scanner;
 
-
 public class GoogleDistanceMatrixAPI {
 
   public static void main(String[] args) throws JSONException {
+
+    // Prompt the user to enter the pickup and dropoff locations
     Scanner scanner = new Scanner(System.in);
     System.out.println("Enter pickup location:");
```
Comment (BaraahAlKalbani) [inline src/main/java/org/example/GoogleDistanceMatrixAPI.java:18]: use the main class (args) instead of getting the value from the user

**Review by BaraahAlKalbani — CHANGES_REQUESTED** [review]
great work But try to change your code according to my previous comment

**Review by BaraahAlKalbani — APPROVED** [review]
Great Work 👍🏻

**Review by WasanMoosa — APPROVED** [review]
Great work!, you can also implement classes, so each one has single responsibility.

**Review by AnwaarAlbusaidi — APPROVED** [review]
well done, try to use SRP.

**Pull request closed without merging**